moc_openstack/manifests/ssl/additional_params.pp

```diff
@@ -0,0 +1,66 @@
+# Apply ssl related changes to controller node
```

Comment:
This file is not used at this moment. All the changes from this file are moved to individual service modules. Keeping those changes here was causing timing issues (like file not present and we trying to modify it).